src/main.rs

```diff
@@ -71,6 +71,13 @@ fn run() -> Result<(), failure::Error> {
                             .help("The ID of the namespace this action applies to")
                             .required(true)
                         )
+                        .arg(
+                            Arg::with_name("force")
```

Comment:
Question: there is also a `--filename / -f` flag for `kv:key put`. Do we want to make this filename flag different as to avoid confusion with the force flag used by all delete subcommands?

Reply:
should we change it to `--path | -p`?